app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    file_to_copy = command[1]
+    new_file = command[-1]
+
+    if file_to_copy != new_file:
+        with open(file_to_copy, "r") as source, open(new_file, "w") as copy:
```

Comment:
Also, I think it would be great to check if our file exists. Maybe use an exception.